py/shared/sql/models.sql

```diff
@@ -92,6 +92,7 @@ CREATE TABLE events (
   start_ts TIMESTAMPTZ NOT NULL,
   timezone VARCHAR(63) NOT NULL DEFAULT 'Europe/London',
   duration INTERVAL,
+  youtube_video_id VARCHAR(140),
```

Comment:
you need to add a patch to `db.py` to add this field.

Reply:
Thanks. Done.